tests/test_formatter/snapshots/snap_test_formatter_output.py

```diff
@@ -107,6 +107,23 @@
 https://wemake-python-stylegui.de/en/xx.xx/pages/usage/violations/
 '''
 
+snapshots['test_formatter[cli_options3-with_links] formatter_with_links'] = '''
+\x1b[4m\x1b[1m./tests/fixtures/formatter/formatter1.py\x1b[0m\x1b[0m
+  1:1      WPS111 Found too short name: s < 2 (https://pyflak.es/WPS111)
+  1:7      WPS110 Found wrong variable name: handle (https://pyflak.es/WPS110)
+  2:21     WPS432 Found magic number: 200 (https://pyflak.es/WPS432)
+  2:21     WPS303 Found underscored number: 2_00 (https://pyflak.es/WPS303)
```

Comment:
Another idea: we can print links on new lines. Why?

1. It maybe more readable and easier to click
2. Chars like `)` won't be considered as a part of the URL by some terminals (hypothetical)

Reply:
Something along the lines of:

```
  1:1      WPS111 Found too short name: s < 2 
           -> https://pyflak.es/WPS111
  1:7      WPS110 Found wrong variable name: handle (https://pyflak.es/WPS110)
           -> https://pyflak.es/WPS110
  2:21     WPS432 Found magic number: 200 (https://pyflak.es/WPS432)
           -> https://pyflak.es/WPS432
  2:21     WPS303 Found underscored number: 2_00 (https://pyflak.es/WPS303)
           -> https://pyflak.es/WPS303
```

?

Also, would it play well with `--show-source`?

Reply:
Let's add a test for it.

What do you think should go first?

Reply:
I've tried both styles and `line, source, link` looks a bit neater IMO:

```
wemake_python_styleguide/formatter.py

  98:77    C812  missing trailing comma
  row_col='{0}:{1}'.format(error.line_number, error.column_number)
                                                                  ^
  -> https://pyflak.es/C812

  121:1    D202  No blank lines allowed after function docstring
  """Called when ``--show-violation-links`` option is provided."""
  ^
  -> https://pyflak.es/D202
```

Implemented this one. For comparison:

<details><summary><code>line, link, source</code></summary>

```
wemake_python_styleguide/formatter.py

  98:77    C812  missing trailing comma
           -> https://pyflak.es/C812
  row_col='{0}:{1}'.format(error.line_number, error.column_number)
                                                                  ^

  121:1    D202  No blank lines allowed after function docstring
           -> https://pyflak.es/D202
  """Called when ``--show-violation-links`` option is provided."""
  ^
```

</details>

Reply:
I really like the second version. This way we have separate spacing for "meta" information and separate spacing for "source code".

Reply:
Can do.